lab3.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int search (int a[],int n) {
+    
+    int b=0;
+    for (int i=0; i<n; i++) {
+        b=b^a[i];    
```

Comment:
what does this line do? 
```suggestion
        b = b ^ a[i];    
```